examples/classification/main.py

```diff
@@ -479,6 +511,9 @@ def validate(val_loader, model, criterion, config):
             config.tb.add_scalar("val/loss", losses.avg, len(val_loader) * config.get('cur_epoch', 0))
             config.tb.add_scalar("val/top1", top1.avg, len(val_loader) * config.get('cur_epoch', 0))
             config.tb.add_scalar("val/top5", top5.avg, len(val_loader) * config.get('cur_epoch', 0))
+            mlflow.log_metric("val/loss", float(losses.avg), step=len(val_loader) * config.get('cur_epoch', 0))
```

Comment:
let's add best_acc1 to `log_param`.

we can do it at the very end of the training, but ideally to override the logged `best_acc1` value on each epoch, if mlflow allows it

Reply:
Done